src/ring/middleware/json.clj

```diff
@@ -118,7 +127,8 @@
   Accepts the following options:
 
   :pretty            - true if the JSON should be pretty-printed
-  :escape-non-ascii  - true if non-ASCII characters should be escaped with \\u"
+  :escape-non-ascii  - true if non-ASCII characters should be escaped with \\u
+  :stream            - true if JSON body should be streamed rather than a string"
```

Comment:
Change this key to: `:stream?`.

Reply:
No problem, was just staying consistent with other opts.

Reply:
Yep, that's understandable. Those options in particular are passed through to Cheshire, which is why they're named that way, rather than being consistent with the rest of the code.